content/sklearn/concepts/nearest-neighbors/nearest-neighbors.md

```diff
@@ -0,0 +1,117 @@
+
+## Applications:
+
+The `sklearn.neighbors` functions include nearest neighbors algorithms for both [supervised and unsupervised learning methods](https://scikit-learn.org/stable/modules/neighbors.html#neighbors). Supervised nearest neighbors are used for classification and regression, while unsupervised nearest neighbors are used for clustering. These algorithms require defining the number of points or neighbors (`n_neighbors`) and the search algorithm (`algorithm`). There are other parameters specific to each method, which you can find and learn more about by checking the [documentation](https://scikit-learn.org/stable/modules/neighbors.html#neighbors).
+
+  - **Nearest Neighbors Classification** predicts the class of a new data point by (1) identifying the nearest neighbors of the new point from the training dataset based on a chosen distance metric, (2) counting how many of these neighbors belong to each class, and (3) assigning the new point to the class that has the most representatives among the neighbors. For example, if the new point has 1 neighbor of class A, 1 neighbor of class B, and 3 neighbors of class C, the new point will be classified as class C because it has the majority vote. `scikit-learn` offers two different classifiers, `KNeighborsClassifier` and `RadiusNeighborsClassifier`. `KNeighborsClassifier` implements learning based on the nearest neighbors of each new point, where the number of neighbors is an integer value specified by the user. `RadiusNeighborsClassifier` implements learning based on the number of neighbors within a fixed radius of each training point, where the radius is a floating-point value specified by the user. The `KNeighborsClassifier` is the most commonly used method. In cases where data is not uniformly sampled,  `RadiusNeighborsClassifier` can be a better choice; however, it is less effective for high-dimensional parameter problems. Basic nearest neighbors classification uses uniform `weights`; however, in some cases, it is recommended to weight the neighbors so that nearer neighbors contribute more to the prediction. 
+
+  - **Nearest Neighbors Regression** predicts the value of a new data point in cases where the data labels and values are continuous (not discrete) by (1) identifying the nearest neighbors of the new point from the training dataset based on a chosen distance metric, (2) averaging the values of these neighbors, and (3) assigning the new point the averaged value. For example, if the new point has 3 neighbors with values 10, 20, and 30, the new point's predicted value will be the average of these, i.e., 20.0. The `scikit-learn` package offers two different regressors: `KNeighborsRegressor` and `RadiusNeighborsRegressor`. Their learning methods are similar to those mentioned in the classification above. 
+
+
+## Codebyte Example 
+
+Here are two simple examples of classification and regression using `KNeighborsClassifier` and `KNeighborsRegressor` from scikit-learn.
```

Comment:
This is incorrect, the entry is on the `NearestNeighbors` term. This is where we can import it from:
```
from sklearn.neighbors import NearestNeighbors
```

Reply:
You will have to change the codes for this entry since the issue is for the from `NearestNeighbors`, the Classifier and Regressor will be other term entries under this.